unlock-app/src/config.js

```diff
@@ -56,7 +56,7 @@ export default function configure(
   const env = runtimeConfig.unlockEnv
 
   let providers = {}
-  let isRequiredNetwork = () => false
+  let isRequiredNetwork = networkId => networkId === 1984
```

Comment:
Can you leave this and instead update the test and Dev environment versions?

Reply:
Done!